pipelines/jobs/configurations/jdk11u_prototype.groovy

```diff
@@ -0,0 +1,35 @@
+targetConfigurations = [
```

Comment:
Are we sure we should be splitting these out to 'prototype'?  I thought some of them were stable/production

Reply:
I think this is up to a high level discussion (maybe in PMC call) of what should be in "prototype" and what remains in "nightly"

from the comments in https://github.com/adoptium/ci-jenkins-pipelines/issues/473#issuecomment-1321764328
I proposed to move `non-temurin variant` and `not ready to be in nightly `(aarch64windows + riscv64)  to "prototype". 
So the weekly-prototype-openjdk-pipeline trigger prototype-openjdk-pipeline to only run AQA(weekly extended) but no JCK
and we only focus the build/test result of "nightly" and "weekly"

Reply:
i've reverted back the config in "nightly" and set "prototype" to empty.
created https://github.com/adoptium/ci-jenkins-pipelines/issues/543  for further discussion what should be in "prototype".